docs/hugo/content/guide/breaking-changes/_index.md

```diff
@@ -23,7 +23,8 @@ Breaking changes are:
 * Alpha CRD versions have been removed
 * Structure change for ResourceGroup status
 
-For more information see [v2.0.0 Breaking Changes](./breaking-changes-v2.0.0).
+<!-- markdown-link-check-disable-next-line -->
```

Comment:
minor: Leave a comment with all of these that has a short explainer (or a link to the PR that explains?)

So that subsequent readers understand why these are ignored?

Reply:
Added.